### DISCUSSION TITLE
from graia.application import GraiaMiraiApplication, Session出现错误

### DISCUSSION THREAD

**Discussion by 3akura** [GraiaProject/Application#160] [Q&A]
报错内容：
Traceback (most recent call last):
  File "D:/qqbot/qqbotpy/qqbot.py", line 2, in <module>
    from graia.application import GraiaMiraiApplication, Session
  File "D:\python3\lib\site-packages\graia\application\__init__.py", line 14, in <module>
    import graia.application.event.mirai  # for init events
  File "D:\python3\lib\site-packages\graia\application\event\mirai.py", line 1, in <module>
    from typing import Any, Dict, Literal, NoReturn, Optional
ImportError: cannot import name 'Literal' from 'typing' (D:\python3\lib\typing.py)

报错的语句是mirai.py文件中的：“from typing import Any, Dict, Literal, NoReturn, Optional”，Any, Dict, NoReturn, Optional都在typing.py文件中，都正常，Literal在typing.pyi文件中，import失败

应该怎么解决

**Comment by 3akura** [comment]
python是3.7

**Comment by 3akura** [comment]
已经解决了

**Comment by GanLuqiao0** [comment]
我也遇到这个问题了，怎么解决的？

**Reply by 3akura** [reply]
版本问题，不要用最新版，建议graia用0.19x，broadcast用0.8x
